src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx

```diff
@@ -242,7 +242,7 @@
     <comment>Description for Formula</comment>
   </data>
   <data name="FormulaPortDataResultToolTip" xml:space="preserve">
-    <value>Result of math computation</value>
+    <value>String representation of the object.</value>
```

Comment:
```suggestion
    <value>Result of math computation</value>
```

Reply:
see this...
https://github.com/martinstacey/Dynamo/pull/4#discussion_r482993667

Reply:
All changes were made using new PR instead of addressing them directly on github